index.bs

```diff
@@ -281,7 +281,8 @@ The API is defined by the following Web IDL fragment.
     interface WebAuthentication {
         Promise < ScopedCredentialInfo > makeCredential (
             Account                                 accountInformation,
-            sequence < ScopedCredentialParameters > cryptoParameters,
+            sequence < CredentialType >             supportedTypes,
+            sequence < AlgorithmIdentifier >        supportedAlgorithms,
```

Comment:
Does this now mean that the RP must be prepared to support all possible combinations of (type, algorithm) from these two lists? It did not have to do that before.

So e.g. if a new type gets defined next year, and a new algorithm the year after that, the RP would have to backport the new algorithm to old types as well?